src/html/partials/concept.html

```diff
@@ -1,4 +1,73 @@
+  </div>
+
+  <ul class="concept-section__list">
+    <li class="concept-section__item">
+      <div class="concept-section__item__card">
+        <h3>She Gets To Know You.</h3>
+        <p>Florence strikes up a conversation, learning details about your life, health, goals, and preferences in an engaging, human way.</p>
+        <p>This conversation unfolds over time, making this feel less like twenty questions and more like the start of a relationship.</p>
+      </div>
+      <img src="../../images/sms/Text_SMS_1.png" alt="Hi! I'm Florence. What's your name?">
+    </li>
+
+    <li class="concept-section__item">
+      <div class="concept-section__item__card">
+        <h3>She Keeps Track of Your Regimen.</h3>
+        <p>Florence helps you remember what to take and when. Image recognition makes adding your medication as easy as taking a picture.</p>
+      </div>
+      <img src="../../images/sms/Text_SMS_2.png" alt="Let's get started adding your medications.">
+    </li>
+    
+    <li class="concept-section__item">
+      <div class="concept-section__item__card">
+        <h3>She Checks Up On You.</h3>
+        <p>As you start taking new medications, Florence will follow up to see if you’re experiencing side effects.</p>
+        <p>She can offer additional information to help you understand if your experience is typical—or if you should talk with your doctor.</p>
+      </div>
+      <img src="../../images/sms/Text_SMS_3.png" alt="It'sbeen three days since you started taking Lisinopril. How're you feeling?">
+    </li>
+    
+    <li class="concept-section__item">
+      <div class="concept-section__item__card">
+        <h3>She Helps You Remember.</h3>
+        <p>With complex regimens, days often blur together. Florence will show you what you’ve taken today—and what’s left to take.</p>
+      </div>
+      <img src="../../images/sms/Text_SMS_4.png" alt="Here's what I have for you today.">
+    </li>
+    
+    <li class="concept-section__item">
+      <div class="concept-section__item__card">
+        <h3>She Finds You Deals.</h3>
+        <p>Because Florence knows your regimen, she can find you ways to reduce the cost of care.</p>
+      </div>
+      <img src="../../images/sms/Text_SMS_5.png" alt="BTW, Walmart is now offering $4 generics">
+    </li>
+    
+    <li class="concept-section__item">
+      <div class="concept-section__item__card">
+        <h3>She Cheers You On.</h3>
+        <p>Florence can send you occasional notes of encouragement to keep you motivated.</p>
+        <p>For patients with asymptomatic conditions, this offers a sense of progress that might otherwise go unnoticed.</p>
+      </div>
+      <img src="../../images/sms/Text_SMS_6.png" alt="Hey Mike! Nice job, your cholesterol is down 20%.">
```

Comment:
Maybe this is off topic, and I'd like to get others opinions, but I like to include additional context for alt tags. Like maybe adding some text at the beginning:

`"Florence might reply with: Hey Mike! Nice job...."`

Reply:
Jeremy and I were discussing this too. Will make those changes where it would make sense in understanding the content as a whole.